### ISSUE TITLE
Report successful build TBTA with Visual Studio 2012 but not VS Community 2015

### ISSUE THREAD

**Issue by JChungYS** [JChungYS/TBTA#2]
A collaborator, csharpener, successfully build from the source with VS 2012, I was not able to do so with VS Community 2015. One of the issues can be about the "quotation marking".
I was cloning the project to another local repo, after changing WINVER and changing the project script to NON-unicode, there are issues:

Severity	Code	Description	Project	File	Line
Error	C2039	'SetWindowTextW': is not a member of 'CWnd'	TA1	c:\users\jeremiah\source\repos\vstbta\aboutdlg.cpp	49
Severity	Code	Description	Project	File	Line
Error	C1003	error count exceeds 100; stopping compilation	TA1	c:\users\jeremiah\source\repos\vstbta\agreementruledlg.cpp	3558
Severity	Code	Description	Project	File	Line
Error	C2001	newline in constant	TA1	c:\users\jeremiah\source\repos\vstbta\analysisinputtextdlg.cpp	98

I tried change TA1 Properties to MultiBye charater set, there are still 4809 Errors, used to be 5000+.

**Comment by todallman** [comment]
Yes, I'd like to port TBTA from VS2012 to a more recent environment (e.g., VS Community 2015), but there are thousands of issues, as this person reported.  These issues can certainly be addressed, but I can't spend the time on that these days.  
Thanks,Tod
